Docs 9782 gaps <- (wrong Jira number)

paigehargrave opened the pull request:
Is any additional info needed for 2a in https://www.mulesoft.org/jira/browse/SE-9782? Also, I was unable to find the broken link mentioned in point 5.

Commit:
Add uninstall info

Commit:
Added uninstall topic

Commit:
Removed wildcard characters

Commit:
Added IOPS details

Commit:
Add new content

Commit:
Add replace license file

Commit:
Add regain Ops Center access info

Commit:
Limitation added for DOCS-5312

Commit:
Added node recovery info

Commit:
Details added for DOCS-3693

Commit:
Added RTF core requirements

sundippatel commented on modules/ROOT/pages/enable-inbound-traffic.adoc:
```diff
@@ -140,6 +140,11 @@ If your secrets group does not appear, ensure you have the *Manage Runtime Fabri
 . Click *Deploy* to deploy the internal load balancer. If successful, a message appears in the bottom-right of the page. The deployment may take up to a minute. You can view the deployment status at thw top of the form. When the status transitions to *Applied*, the internal load balancer is successfully deployed and inbound traffic enabled.
 
+[IMPORTANT]
+When deploying a mule application to Runtime Fabric through the Runtime Manager UI, the Runtime Manager always 
```
**M**ule.
Remove "the" when referring to Runtime Manager.

sundippatel replied:
This can be reworded for better public consumption; something along these lines:

"When deploying a Mule application to Runtime Fabric using the Runtime Manager UI, only the first domain used in the internal load balancer's certificate is selected to create the application endpoint. Other domains specified in the certificate via the Subject Alternative Name or Common Name are not used."

paigehargrave replied:
Updated this - thank you!

sundippatel commented on modules/ROOT/pages/install-sys-reqs.adoc:
```diff
@@ -16,20 +18,33 @@ Anypoint Runtime Fabric requires one of the following operating systems.
 
-Use the same operating system for each VM. If you attempt to install Runtime Fabric on another operating system version or distribution, the Runtime Fabric installer fails.
+Use the same operating system for each VM. If you attempt to install Runtime Fabric on another operating system version 
+or distribution, the Runtime Fabric installer fails.
```
consider: ".. Fabric installer will fail."

paigehargrave replied:
Fixed this.

sundippatel commented on modules/ROOT/pages/install-sys-reqs.adoc:
```diff
@@ -16,20 +18,33 @@ Anypoint Runtime Fabric requires one of the following operating systems.
+or distribution, the Runtime Fabric installer fails.
 
+== Hardware Requirements
+Runtime Fabric reserves a fixed amount of CPU, 0.3 cores per worker VM, for internal services. This is an important factor when determining the amount of CPU cores and worker nodes needed.
```
As a consistency cleanup task, I think we should sub out "VM" for "node" when referring to either in a more general sense (in other words, when not outlining a requirement). Happy to discuss.

paigehargrave replied:
Agreed.

sundippatel commented on modules/ROOT/pages/install-sys-reqs.adoc:
```diff
@@ -16,20 +18,33 @@ Anypoint Runtime Fabric requires one of the following operating systems.
+or distribution, the Runtime Fabric installer fails.
 
+== Hardware Requirements
+Runtime Fabric reserves a fixed amount of CPU, 0.3 cores per worker VM, for internal services. This is an important factor when determining the amount of CPU cores and worker nodes needed.
```
I think we can streamline the first sentence: "By default, Runtime Fabric reserves 0.3 CPU cores per worker node for internal services."

paigehargrave replied:
Yup - agreed - fixed.

sundippatel commented on modules/ROOT/pages/install-sys-reqs.adoc:
```diff
@@ -16,20 +18,33 @@ Anypoint Runtime Fabric requires one of the following operating systems.
+VMs, each with 2 cores, Runtime Fabric reserves a total of 0.9 cores (0.3 x 3). The number of available cores displayed 
+in the Runtime Manager is 5.1 instead of the full 6 cores.
+
+Overprovisioning RTF worker nodes is recommended to preserve uptime in the event a node becomes unavailable or during
```
"RTF" -> "Runtime Fabric"

paigehargrave replied:
Good catch - thank you!

sundippatel commented on modules/ROOT/pages/install-sys-reqs.adoc:
```diff
@@ -16,20 +18,33 @@ Anypoint Runtime Fabric requires one of the following operating systems.
 === Controller VMs
 
-The minimum number of controller VMs in a production environment is three. The maximum is five, and even numbers of controllers are not supported. These requirements reflect the minimum requirements to maintain high-availability and ensure system performance.
+The minimum number of controller VMs in a production environment is three. The maximum is five, and even numbers of
```
There's a known limitation identified where the max number of controller nodes is 3; thus, we can simplify this to read: "The number of controller nodes required for a production environment is three. Even numbers of controller nodes are not supported."...

paigehargrave replied:
Updated - thank you!

sundippatel commented on modules/ROOT/pages/install-sys-reqs.adoc:
```diff
@@ -54,29 +76,36 @@ Each worker VM must have the following:
 image::architecture-development.png[]
 
-The following sections describe the minimum system requirements of each controller and worker VMs in a development configuration.
+The following sections describe the minimum system requirements of each controller and worker VMs in a 
```
VMs -> VM (or node)

paigehargrave replied:
Updated - thank you!

sundippatel commented on modules/ROOT/pages/install-sys-reqs.adoc:
```diff
@@ -54,29 +76,36 @@ Each worker VM must have the following:
 === Controller VM
 
-Runtime Fabric requires at least one controller VM in a development environment. The maximum supported is five, and even numbers of controllers are not supported.
+Runtime Fabric requires at least one controller VM in a development environment. The maximum supported is five, and 
```
five -> three (per earlier comment)

paigehargrave replied:
Updated.

sundippatel commented on modules/ROOT/pages/replace-license-key.adoc:
```diff
@@ -0,0 +1,45 @@
+= Replace a Mule License Key
```
This is outdated now -- we recommend customers use "rtfctl" to update their Mule license key.

paigehargrave replied:
Updated this - please make sure there is enough info.

sundippatel replied:
Added in a bit. Please have a look.

paigehargrave replied:
LGTM

sundippatel commented on modules/ROOT/pages/troubleshoot-guide.adoc:
```diff
@@ -43,6 +43,26 @@ The Anypoint Monitoring agent may also change the state of a deployed applicatio
+
+Node is down
+
+CRITICAL / Kubernetes node is not ready: 168.189.5.22
```
should prune out the IP address, as that'll vary.

paigehargrave replied:
Updated.

sundippatel commented on modules/ROOT/pages/troubleshoot-guide.adoc:
```diff
@@ -43,6 +43,26 @@ The Anypoint Monitoring agent may also change the state of a deployed applicatio
+$ tsh --cluster=production ssh admin@node gravity status
+----
+
+To recover a node, follow the instructions provided in https://gravitational.com/gravity/docs/cluster/#recovering-a-node[Recovering a Node, role=external, window=_blank].
```
We don't want to point customers to gravitational docs -- there are some things they offer (or may offer in future) which we may decide to not support.

paigehargrave replied:
@sundippatel - So this is an interesting one - the recover instructions in the gravitational docs look like removing and re-adding a node. Is this what we want, or should we be providing instructions on restarting a node?

sundippatel replied:
We'd want instructions on recovering the node.

We have an existing topic "Add and Remove a Node" which is based off this link from Gravitational's docs. Perhaps we can just sub out the links for our own page.

paigehargrave replied:
@sundippatel  Added link to our info - PTAL - thank you!

sundippatel commented on modules/ROOT/pages/troubleshoot-guide.adoc:
```diff
@@ -43,6 +43,26 @@ The Anypoint Monitoring agent may also change the state of a deployed applicatio
+----
+Run the `gravity status` command to obtain the health status of the cluster as well as individual components. For example, to remotely log in to a cluster and check the status of the cluster named `production`, run the following command:
+----
+$ tsh --cluster=production ssh admin@node gravity status
```
Question -- where did these instructions come from? We could simply ask the user to open a terminal to one of their Runtime Fabric controller nodes and run "gravity status".

This assumes the customer has tsh installed on their local machine, and I don't think we mention using that tool elsewhere.

paigehargrave replied:
Updated this.

sundippatel commented on modules/ROOT/pages/uninstall-manual.adoc:
```diff
@@ -0,0 +1,60 @@
+. On each Worker VM, run the following command and confirm by typing “yes” when prompted:
++
+----
+# Leave the Runtime Fabric and remove services
```
Can probably remove out the comment in examples like these

paigehargrave replied:
Removed comments.

sundippatel commented on modules/ROOT/pages/using-opscenter.adoc:
```diff
@@ -33,6 +33,15 @@ cat /var/log/rtf-init.log | grep -B 1 -A 3 "Ops Center"
+== Regain Access to the Ops Center
+If you lose the install logs, perform the following steps:
+
+. SSH into one of the nodes of the Runtime Fabric installation.
```
May be able to simplify: "Open a terminal into one of the controller nodes."

paigehargrave replied:
Updated.

paigehargrave replied:
Got it - thanks!

sundippatel reviewed (changes requested):
Great stuff in this one.

Commit:
Update per Sundip

Commit:
Updated per Sundip

Commit:
Updates per Sundip

Commit:
Removed comments from examples

Commit:
Update per Sundip and spacing updates

Commit:
Updated per Sundip

sundippatel committed:
Update enable-inbound-traffic.adoc

Minor typo

sundippatel committed:
Update replace-license-key.adoc

Added command to set mule license.
Added "See Also" section linking on how to install rtfctl utility.

sundippatel commented:
@paigehargrave Looks great. Left one answer and further clarified one topic for your review.

Commit:
Typo

Commit:
Updated link to add/remove node info

Commit:
Merge branch 'v1.3' into DOCS-9782-gaps-ph
